games-action/nanosaur/nanosaur-1.4.3.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="Vintage 3D dinosaur shooter game"
+DESCRIPTION="This is Pangea Software’s Nanosaur updated to run on modern systems. Originally \
+released in 1998 as a Mac-only game, Nanosaur features a cybernetic dinosaur from the future \
+sent back in time 20 minutes before a giant asteroid hits the Earth."
+HOMEPAGE="https://pangeasoft.net/nano"
+COPYRIGHT="1998 Pangea Software, Inc."
+LICENSE="CC-BY-NC-SA-4.0"
+REVISION="1"
+
+SOURCE_URI="https://github.com/jorio/Nanosaur/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="5458ed6bf7d58facd7adf30161680546148b2a4e187c336b8af02cb864188da0"
+SOURCE_DIR="Nanosaur-$portVersion"
+PATCHES="nanosaur-$portVersion.patchset"
+ADDITIONAL_FILES="nanosaur.rdef.in"
+srcGitRev2="0581b241624d16c453e0afab57a379b443229260"
+SOURCE_URI_2="https://github.com/jorio/Pomme/archive/$srcGitRev2.tar.gz"
+CHECKSUM_SHA256_2="5e74b0061908665e7a9c710d83969aa0234b78766b05eacfa86cc3c1c0d1f666"
+SOURCE_FILENAME_2="Pomme-$srcGitRev2.tar.gz"
+SOURCE_DIR_2="Pomme-$srcGitRev2"
+PATCHES_2="nanosaur-$portVersion-source2.patchset"
+
+ARCHITECTURES="all ?x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	nanosaur = $portVersion
+	app:Nanosaur = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl2_2.0$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	# submodule
+	rm -rf extern/Pomme
+	ln -s $sourceDir2 extern/Pomme
+	# building
+	cmake -S . -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo
+	cmake --build build -j $jobArgs
```

Comment:
```suggestion
	cd build 
	make $jobArgs
```
need to workaround haikuporter being too strict about calling cmake witough CMAKE_BUILD_TYPE